docs/data-cookbooks/cnm-workflow.md

```diff
@@ -0,0 +1,475 @@
+# CNM Workflow
+
+This entry documents setup of a basic workflow that utilizes the built-in CNM/Kinesis functionality in Cumulus.
+
+Prior to working through this entry you should be familiar with the [Cloud Notification Mechanism](https://wiki.earthdata.nasa.gov/display/CUMULUS/Cloud+Notification+Mechanism).
```

Comment:
We don't really have a good referent link for this until the CNM documentation goes 'public', and this content is somewhat out of date relevant to the draft doc copy we have.        I'm thinking we should post the draft .doc internally and link there until the ICD is approved.   Thoughts?

Reply:
I think this goes back to my questions about whether the data cookbook itself was appropriate for general open source with the rest of the cumulus docs.  It's pretty DAAC-specific in places.  I think the link you have is probably the right answer.

Reply:
I think I can get behind using the link, but we should include a task so we don't lose track of updating it once the ICD is complete, wherever this doc eventually winds up.

Reply:
I've added CUMULUS-827 to track that chore.